app/src/main/java/org/oppia/android/app/onboardingv2/CreateProfileFragmentPresenter.kt

```diff
@@ -0,0 +1,122 @@
+package org.oppia.android.app.onboardingv2
+
+import android.app.Activity
+import android.content.Intent
+import android.graphics.PorterDuff
+import android.graphics.drawable.Drawable
+import android.net.Uri
+import android.provider.MediaStore
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.res.ResourcesCompat
+import androidx.fragment.app.Fragment
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.DataSource
+import com.bumptech.glide.load.engine.GlideException
+import com.bumptech.glide.request.RequestListener
+import com.bumptech.glide.request.RequestOptions
+import com.bumptech.glide.request.target.Target
+import org.oppia.android.R
+import org.oppia.android.app.fragment.FragmentScope
+import org.oppia.android.databinding.CreateProfileFragmentBinding
+import javax.inject.Inject
+
+private const val GALLERY_INTENT_RESULT_CODE = 1
+
+/** Presenter for [CreateProfileFragment]. */
+@FragmentScope
+class CreateProfileFragmentPresenter @Inject constructor(
+  private val fragment: Fragment,
+  private val activity: AppCompatActivity,
+  private val createProfileViewModel: CreateProfileViewModel
+) {
+  private lateinit var binding: CreateProfileFragmentBinding
+  private lateinit var uploadImageView: ImageView
+  private var selectedImage: Uri? = null
+
+  /** Initialize layout bindings. */
+  fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?): View {
+    binding = CreateProfileFragmentBinding.inflate(
+      inflater,
+      container,
+      /* attachToRoot= */ false
+    )
+    binding.let {
+      it.lifecycleOwner = fragment
+      it.viewModel = createProfileViewModel
+    }
+
+    uploadImageView = binding.createProfileUserImageView
+    Glide.with(activity)
```

Comment:
Do we need to do this? It'd be much preferred to use our existing utilities to load images via Glide as, otherwise, this can complicate our Glide management.

Ditto elsewhere.

Reply:
Switched to our utilities.